tests/e2e/pages/landingPage.js

```diff
@@ -0,0 +1,65 @@
+exports.LandingPage = class LandingPage {
+  constructor(page) {
+    this.page = page;
+    this.loginButton = page.locator('#sign-in-btn');
+    this.vpnBanner = page.locator('.vpn-banner');
+    this.header = page.locator('#navigation-wrapper section').first();
+    this.monitorLanding = page.locator('.monitor-landing .landing-content');
+    this.personalDataBanner = page.locator('.take-back-control-banner');
+    this.homePageNavButton = page.locator(
+      '.desktop-menu [data-event-label="Home"]'
+    );
+    this.breachesPageNavButton = page.locator(
+      '.desktop-menu [data-event-label="Breaches"]'
+    );
+    this.secTipsNavButton = page.locator(
+      '.desktop-menu [data-event-label="Security Tips"]'
+    );
+    this.checkForBreachesEmailInput = '#scan-email';
+    this.CheckForBreachesButton = page.locator(
+      '#scan-user-email [data-entrypoint="fx-monitor-check-for-breaches-blue-btn"]'
+    );
+    this.newsLetterCheckBox = page.locator('.create-fxa-checkbox-checkmark');
+    this.breachCard = page.locator('.latest-breach-info');
+    this.monitorAccountFeatures = page.locator(
+      '(//section[@id="fx-account-features"]//div)[4]'
```

Comment:
I believe id attributes pollute global scope – unless it's important for something to have an id,  I'd prefer class, in particular if the element is styled.  

Really this will all be redundant as we refactor/refresh.  Best not to spend too much time right now